data_model/src/events/time.rs

```diff
@@ -408,10 +408,7 @@ mod tests {
                 Schedule::starting_at(Duration::ZERO).with_period(Duration::from_millis(1));
             let interval = Interval::new(Duration::from_secs(TIMESTAMP), Duration::from_secs(1));
 
-            assert_eq!(
-                EventFilter::count_matches_in_interval(&schedule, &interval),
-                1000
-            );
+            EventFilter::count_matches_in_interval(&schedule, &interval);
```

Comment:
This is not what I expected.
If we have a bench, let's make it a bench instead of a `#[test]`

Reply:
Yeah, I got your suggestion wrong. I'm working on making it bench now

Reply:
@s8sato , can you check new commit, please?